### PR TITLE
SC-16970 add release notes for sematext-agent 3.1.0

### PR DESCRIPTION
**Pull request by boratanrikulu**

### CODE REVIEW

**Review by regiluze — APPROVED** [review]
LGTM

File: docs/agents/sematext-agent/releasenotes.md [hunk 1]
```diff
@@ -35,6 +35,28 @@ NULL
...
 -->
 
+## Version 3.1.0
+
+Date: August 24, 2023
+
+### New Features
+
+- Varnish Cache metrics monitoring support: With this update, Sematext Agent can now discover and monitor your Varnish Cache service. Stay tuned for the upcoming release of Varnish Cache monitoring integration in Sematext Cloud.
+- Now supporting for kube-proxy and API server monitoring in Google Kubernetes Engine (GKE) as part of our Kubernetes monitoring integration.
```
Comment (cpipilas) [inline docs/agents/sematext-agent/releasenotes.md:45]: I would prefer "Support has been added for kube-proxy and API server monitoring in Google Kubernetes Engine (GKE) as part of our Kubernetes monitoring integration."

File: docs/agents/sematext-agent/releasenotes.md [hunk 2]
```diff
@@ -35,6 +35,28 @@ NULL
 
 -->
 
+## Version 3.1.0
+
+Date: August 24, 2023
+
+### New Features
+
+- Varnish Cache metrics monitoring support: With this update, Sematext Agent can now discover and monitor your Varnish Cache service. Stay tuned for the upcoming release of Varnish Cache monitoring integration in Sematext Cloud.
```
Comment (cpipilas) [inline docs/agents/sematext-agent/releasenotes.md:44]: I think it's better to replace with: ...Sematext Agent can now discover **and collect metrics** from your ....

File: docs/agents/sematext-agent/releasenotes.md [hunk 3]
```diff
@@ -35,6 +35,28 @@ NULL
...
+- Now supporting for kube-proxy and API server monitoring in Google Kubernetes Engine (GKE) as part of our Kubernetes monitoring integration.
+
+### Improvements
+
+- We've fine-tuned data collection in the Kubernetes monitoring integration, resulting in improved performance and efficiency.
+- Improved discovery for ElasticSearch integration: We have enhanced the ElasticSearch integration by preventing the discovery of `.log` files, which are not supported. Now, Sematext Agent will only discover and support `.json` files for ElasticSearch integration, ensuring a smoother experience.
```
Comment (cpipilas) [inline docs/agents/sematext-agent/releasenotes.md:50]: remove "ensuring a smoother experience.".

File: docs/agents/sematext-agent/releasenotes.md [hunk 4]
```diff
@@ -35,6 +35,28 @@ NULL
...
+- Introducing unified runtime metrics (goCollector) to provide a comprehensive view of system performance in the Kubernetes monitoring integration.
+- Enriched Vector diagnostics information for baremetal, Kubernetes, and Docker setups, facilitating better troubleshooting.
```
Comment (cpipilas) [inline docs/agents/sematext-agent/releasenotes.md:52]: remove " facilitating better troubleshooting."

File: docs/agents/sematext-agent/releasenotes.md [hunk 5]
```diff
@@ -35,6 +35,28 @@ NULL
...
+
+### Bug Fixes
+
+- Rectified incorrect expected and live instances metrics in the Kubernetes monitoring integration.
```
Comment (cpipilas) [inline docs/agents/sematext-agent/releasenotes.md:57]: "Corrected inaccurate expected and live instance metrics within the Kubernetes monitoring integration."

File: docs/agents/sematext-agent/releasenotes.md [hunk 6]
```diff
@@ -35,6 +35,28 @@ NULL
 
 -->
 
+## Version 3.1.0
+
+Date: August 24, 2023
+
+### New Features
+
+- Varnish Cache metrics monitoring support: With this update, Sematext Agent can now discover and and collect metrics from your Varnish Cache service. Stay tuned for the upcoming release of Varnish Cache monitoring integration in Sematext Cloud.
```
Comment (yago-123) [inline docs/agents/sematext-agent/releasenotes.md:44]: `and and`

Comment (cpipilas) [inline docs/agents/sematext-agent/releasenotes.md:44]: Varnish Cache metrics monitoring -> Varnish Cache monitoring

File: docs/agents/sematext-agent/releasenotes.md [hunk 7]
```diff
@@ -35,6 +35,28 @@ NULL
...
+- Support has been added for kube-proxy and API server monitoring in Google Kubernetes Engine (GKE) as part of our Kubernetes monitoring integration.
+
+### Improvements
+
+- We've fine-tuned data collection in the Kubernetes monitoring integration, resulting in improved performance and efficiency.
+- Improved discovery for ElasticSearch integration: We have enhanced the ElasticSearch integration by preventing the discovery of `.log` files, which are not supported. Now, Sematext Agent will only discover and support `.json` files for ElasticSearch integration.
+- Enriched Vector diagnostics information for baremetal, Kubernetes, and Docker setups.
+- Introducing unified runtime metrics (goCollector) to provide a comprehensive view of system performance in the Kubernetes monitoring integration. You check all supported runtime metrics in [this wiki](https://sematext.com/docs/integration/kubernetes/#runtime-metrics).
```
Comment (yago-123) [inline docs/agents/sematext-agent/releasenotes.md:52]: goCollector? If it's internal is OK, if is public I would change it a bit before we expose

**Review by yago-123 — CHANGES_REQUESTED** [review]

File: docs/agents/sematext-agent/releasenotes.md (hunk 7 above)
Comment (boratanrikulu) [reply docs/agents/sematext-agent/releasenotes.md:52]: This is public, okay will remove `(goCollector)`

**Review by cpipilas — APPROVED** [review]

**Review by yago-123 — APPROVED** [review]

**Pull request merged**